charts/neuvector/latest/templates/pvc.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if and .Values.controller.enabled .Values.controller.pvc.enabled -}}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: neuvector-data
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ template "neuvector.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  accessModes:
+    - ReadWriteMany
```

Comment:
the PVC accessModes should be configurable, and most storage will be default to `ReadWriteOnce`.

Reply:
In majority of cases, PVC is not needed as we can maintain the state internally. However, if the user does want to enable PVC in order to mitigate the catastrophic failure, we do require ReadWriteMany mode for PVC.